tests/test_ecdsa.py

```diff
@@ -0,0 +1,82 @@
+#! pytest
+
+from web3 import Web3
+
+"""
+NOTE Tests are ported from OpenZeppelin
+(https://github.com/OpenZeppelin/openzeppelin-solidity/blob/master/test/cryptography/ECDSA.test.js)
```

Comment:
Should add the commit hash as a version, and the license (MIT) from OpenZeppelin